src/app/_component/home-page.tsx

```diff
@@ -1,11 +1,13 @@
 import Banner from '@/components/banner/banner';
+import Faq from '@/components/faq/faq';
```

Comment:
No abbreviations, per standard.

Reply:
Addressed in [297792e](https://github.com/OpenBCca/openbc-web/pull/92/commits/297792e8aa6e108354a56e0cebfe14e20f12f1bf)